service/openapi.yaml

```
@@ -0,0 +1,239 @@
+openapi: 3.1.0
+info:
+  version: 0.1.0
+  title: OpenFeature Remote Evaluation Protocol (OFREP)
+  description: OFREP define the protocol for remote flag evaluations
+  contact:
+    url: https://github.com/open-feature/protocol
+  license:
+    identifier: Apache 2.0
```

```suggestion
    identifier: Apache-2.0
```

This must match the SPDX identifier defined [here](https://spdx.org/licenses/).